src/routes/blog/post/dont-blame-the-readers-write-the-docs-they-need/+page.markdoc

```diff
@@ -0,0 +1,111 @@
+difficult for the community to contribute, and incomplete coverage of the discover, evaluate, learn, build, and scale stages of a typical developer journey.
+
+Then, we looked at other docs sites for inspiration. 
+We liked various aspects of [Stripe](https://stripe.com/docs), [React](https://react.dev/learn), [Vue](https://vuejs.org/guide/essentials/reactivity-fundamentals.html), 
+and [Datadog](https://docs.datadoghq.com/) docs, which inspired our design process.
+We notice how their approach to documentation avoids or generates friction points. 
+We took what we liked, and left out what didn’t work for us.
+
+We also attend conferences like DevRelCon and Refactor DX to learn about the latest trends in developer experience, technical writing, and accessibility to evaluate our design's strengths and shortcomings.
+A bonus is the fresh eyes we pull aside every conference for in-person feedback, especially rare for a fully remote team like us.
+
+## Building the new docs
+
+The new Appwrite docs need to cover the entire developer journey. This means it needs to help the developers discover Appwrite, evaluate its capabilities, learn to use Appwrite, build something useful, and scale it to support product growth.
+
+Appwrite’s new documentation has a navigation flow that aims to follow this journey. We introduced overview pages that introduce Appwrite’s capabilities, platform pages to help developers evaluate Appwrite’s design, more quick starts for frameworks to shorten the learning stage and build confidence, and expanded tutorials that help developers build and scale.
+
+![Overview of Appwrite's new documentation](/images/blog/dont-blame-the-readers-write-the-docs-they-need/overview.png)
+
+Developers loved the simplicity of the old Appwrite documentation, and we didn’t want to lose that part of our identity. 
+Each deep dive section still begins with the same one-page quick start, which may be enough information for veterans. 
+More comprehensive documentation and suggested usage are available, but further down the navigation.
+
+![Quick start in the Authentication docs page](/images/blog/dont-blame-the-readers-write-the-docs-they-need/quickstart.png)
+
+For learners and students, we added step by step tutorials to help you get to a working app faster and build confidence.
+
+![A complete tutorial in the new docs](/images/blog/dont-blame-the-readers-write-the-docs-they-need/tutorials.png)
+
+With all the new content, the new Appwrite docs also introduce **search**. Search helps keep Appwrite docs easy to navigate.
+
+![Search feature in new documentation](/images/blog/dont-blame-the-readers-write-the-docs-they-need/search.png)
+
+The new docs are written using an extended flavor of markdown made with [Markdoc and Svelte](https://github.com/TorstenDittmann/svelte-markdoc-preprocess), which makes community contributions easier. Markdown is easy to read and we can extend it with custom syntax like this multi-code selector.
+
+``````
```

Comment:
Why the ticks before the multicode?

Reply:
> Why the ticks before the multicode?

It shows the syntax, we don't actually want a multicore to be displayed.